### PR TITLE
cli: update debug recover command procedure help

### PR DESCRIPTION
**Pull request by aliher1911**
Updated recovery procedure with steps that user need to perform
when executing unsafe loss of quorum recovery.

Release note: None

### CODE REVIEW

File: pkg/cli/debug_recover_loss_of_quorum.go [hunk 1]
```diff
@@ -102,21 +102,27 @@ taken ASAP. Those actions should be done at application level.
 recovery operation. To perform recovery one should perform this sequence
 of actions:
 
-0. Stop the cluster
+0. Decommission failed nodes preemptively to eliminate the possibility of
+to be removed replicas coming back online and conflicting with recovered
```
Comment (erikgrinaker) [inline pkg/cli/debug_recover_loss_of_quorum.go:106]:
```suggestion
them coming back online and conflicting with the recovered
```

File: pkg/cli/debug_recover_loss_of_quorum.go [hunk 2]
```diff
@@ -102,21 +102,27 @@ taken ASAP. Those actions should be done at application level.
...
+state. Note that if system ranges suffer loss of quorum, it may be impossible
+to decommission nodes. In that case, recovery can proceed, but those nodes
+must be decommissioned once cluster is back online after recovery.
```
Comment (erikgrinaker) [inline pkg/cli/debug_recover_loss_of_quorum.go:109]:
```suggestion
must be prevented from communicating with the cluster and must be decommissioned once the cluster is back online after recovery.
```

**Review by erikgrinaker — APPROVED** [review]
LGTM

**Comment by aliher1911** [comment]
bors r=erikgrinaker

**Pull request merged**